src/components/AlertCard/index.tsx

```diff
@@ -1,78 +1,159 @@
+        styles[variant],
+        styles[size],
+        styles[layout],
+        {
+          [styles.noBorderLeft]: noBorderLeft,
+          [styles.iconContainer]: iconContainer
+        },
+        className
+      )}
       {...props}
     >
-      <div className={styles.icon}>{icon ?? <WarningIcon />}</div>
-      <div className={styles.text}>
-        <div className={classNames('button', styles.title)}>{title}</div>
-        <div
-          className={classNames(
-            {
-              'body-small': variant === 'warning',
-              body: variant === 'danger'
-            },
-            styles.message
-          )}
-        >
-          {message}
-        </div>
-      </div>
       {showClose ? (
-        <div className={styles.close}>
-          <CloseButton onClick={onClose} />
-        </div>
+        <button type="button" onClick={onClose} className={styles.closeButton}>
+          {closeButton}
+        </button>
       ) : null}
-      {actionText && (
-        <div className={styles.action}>
-          <Button
-            rightIcon={<RightArrow />}
-            onClick={onActionClick}
-            type="link"
-            size="small"
-          >
-            {actionText}
-          </Button>
+      {showIcon ? <div className={styles.icon}>{icon}</div> : null}
+      <div className={styles.content}>
+        <div className={styles.header}>
+          <div className={styles.title}>{title}</div>
         </div>
-      )}
+        <div className={styles.message}>{message}</div>
+
+        {listItems ? (
+          <ul className={styles.list}>
+            {listItems.map((item, index) => (
+              <li key={index}>{item}</li>
+            ))}
+          </ul>
+        ) : null}
+
+        {link ? (
+          <div className={styles.link}>
+            <Button
+              type="tertiary"
+              size="small"
+              onClick={link.onClick}
+              rightIcon={<RightArrow />}
+            >
+              {link.text}
+            </Button>
+          </div>
+        ) : null}
+
+        {buttons ? (
+          <div className={styles.buttonGroup}>
+            {buttons.primary && (
+              <Button
+                type="primary"
+                size="small"
+                onClick={buttons.primary.onClick}
+              >
+                {buttons.primary.text}
+              </Button>
+            )}
+            {buttons.secondary && (
+              <Button
+                type="secondary"
+                size="small"
+                onClick={buttons.secondary.onClick}
+              >
+                {buttons.secondary.text}
+              </Button>
+            )}
+            {buttons.tertiary && (
+              <Button
+                type="tertiary"
+                size="small"
+                onClick={buttons.tertiary.onClick}
+              >
+                {buttons.tertiary.text}
+              </Button>
+            )}
+          </div>
+        ) : null}
```

Comment:
could we just make this a `React.ReactNode` and let the consuming code pass in whatever buttons they want in a `<></>`? Otherwise we'll have to do some prop drilling here if we want to pass in props on the button component like you are doing here with onClick

Reply:
I had a doubt here, because the buttons here in the alert card must be small, and they must have a standard position, where the primary, secondary and link buttons are (if left or right for example. At least that's what I understood from the ui design and from what I understand about the order of buttons, where the primary, secondary and tertiary buttons should be, but since I'm not working on the design anymore, I think we should clear this doubt with @pimentel.